test/test_tensor.py

```diff
@@ -240,5 +241,14 @@ def test_tensor_copy(self):
     x = copy.deepcopy(Tensor.ones((3,3,3)))
     np.testing.assert_allclose(x.numpy(), np.ones((3,3,3)))
 
+  def test_copy_from_disk(self):
+    raw = np.random.randint(0, 256, size=30, dtype=np.uint8).tolist() # [0, 255
+    with open("/tmp/test_my_binary.bin", "wb") as f:
```

Comment:
This should probably use `temp` like here: https://github.com/nimlgen/tinygrad/blob/4a9cdd9c4ffacda26f537ac2c19d44a4098e4601/test/unit/test_disk_tensor.py#L95

Reply:
done